### ISSUE TITLE
Error nearest measuring station panel

### ISSUE THREAD

**Issue by opeeters** [irceline/aq-mobile-be#163]
A bug was introduced to the nearest measuring station panel (probably with the merge of feature/new-diagram-view branch):

![Screenshot_20190502-133528](https://user-images.githubusercontent.com/323719/57073458-6a65ab00-6ce1-11e9-9153-1ad454fcb8e4.png)

Browser console gives:
ERROR Error: "No value returned"

**Comment by SpeckiJ** [comment]
The backend seems to have changed a bit. Previously the first 9 letters of the Station name supplied by the WFS were cut off (presumably because they contained an internal identifier?). It seems that this cutoff is now not needed anymore and the station name can be used as supplied.

(This explains the missing Station Names).

Additionally it seems like the PM10 and PM25 Requests currently do not produce any output (we only recieve empty FeatureCollections).

(This explains the missing values for the PM Phenomena.

**Comment by SpeckiJ** [comment]
I have refactored the error handling to be more verbose here: https://github.com/irceline/aq-mobile-be/commit/030d49ccd2097d3b922a85f1655f49195f270776

**Comment by opeeters** [comment]
I think I understand what's going wrong. Originally we got the station data exclusively via the [REST-api](http://geo.irceline.be/sos/api/v1/). Since the BelAQI is based on 24hmean's for the PM's, we decided to get that data via the [WFS](http://geo.irceline.be/wfs) instead (since there are no 24hmean time series being served via the REST-api). Via the WFS you have a `pm10_24hmean_station` and `pm25_24hmean_station` layer. 

The station names served via the REST-api start with a station code (indeed, the first 9 digits where striped at some stage since the name was to long for the layout of the old app).

I think that this difference between the PM's and the rest was not properly implemented in the nearest station panel, and now, with the Chart-page being resuscitated, we might as well go back to a REST-api only system and use the `fetch24hMeanData`-function (cf components/belaqi-diagram-slider/belaqi-diagram-slider.component.ts), or something similar.
